### PR TITLE
formatted SDK & added NCW endpoints

### PR DESCRIPTION
**Pull request by yanlevi**

### CODE REVIEW

File: fireblocks_sdk/sdk.py [hunk 1]
```diff
@@ -51,16 +84,177 @@ def __init__(self, private_key, api_key, api_base_url="https://api.fireblocks.io
...
+                "X-API-Key": self.api_key,
+                "User-Agent": self._get_user_agent(anonymous_platform),
+            }
+        )
+        self.NCW = self._create_ncw()
+
+    def _create_ncw(self):
+        return self.NCW(self)
+
+    class NCW:
```
Comment (idanya) [inline fireblocks_sdk/sdk.py:98]: I think we can extract this class to a different file. it gets the SDK as arg, so no real reason to define it under the SDK class. To initiate it you'll still have to initiate the SDK first and pass it in the constructor.

File: fireblocks_sdk/sdk.py [hunk 2]
```diff
@@ -2497,3 +2227,162 @@ def _get_user_agent(anonymous_platform):
                 f"{platform.machine()})"
             )
         return user_agent
+
+
+class FireblocksNCW:
```
Comment (idanya) [inline fireblocks_sdk/sdk.py:2232]: can we extract to a new file?

File: fireblocks_sdk/ncw_sdk.py [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+    def get_wallet_asset_addresses(
+        self,
+        wallet_id: str,
+        account_id: str,
+        asset_id: str,
+        page_cursor: str = None,
+        page_size: int = None,
+        sort: str = None,
+        order: str = None,
+        enabled: bool = None,
```
Comment (idanya) [inline fireblocks_sdk/ncw_sdk.py:121]: if we set to default None, the type should be optional.

**Review by idanya — APPROVED** [review]

File: fireblocks_sdk/sdk.py [hunk 4]
```diff
@@ -1714,77 +2057,6 @@ def get_users(self):
...
-    def delete_user_group(self, id: str) -> None:
-        """
-        Deletes a Users Group
-        @param id: The ID of the Users Group
-        """
-
-        url = f"/v1/users_groups/{id}"
-
-        return self._delete_request(url)
-
```
Comment (YoavBZ) [inline fireblocks_sdk/sdk.py:1787]: Why did you remove the `/users_groups` methods?

Comment (yanlevi) [reply fireblocks_sdk/sdk.py:1787]: It was a mistake, due to the PR existing for a long time, we should put them back

Comment (YoavBZ) [reply fireblocks_sdk/sdk.py:1787]: @yanlevi @idanya Please do that as soon as possible
